internal/cfgplugins/qos.go

```diff
@@ -375,6 +373,8 @@ func GetPolicyCLICounters(t *testing.T, dut *ondatra.DUTDevice, policyName strin
 	switch dut.Vendor() {
 	case ondatra.ARISTA:
 		cliConfig := fmt.Sprintf("show policy-map type qos %s counters", policyName)
+		helpers.GnmiCLIConfig(t, dut, cliConfig)
+		t.Logf("Applied Arista cEOS specific config for %s: %s", policyName, cliConfig)
```

Comment:
Are these CLI specific to EOS ONLY on cEOS DUT?   Or any EOS based DUT?

If only cEOS, then a condition should be added to ensure this is only used on cEOS and not hardware targets.

(for example `if strings.ToLower(dut.Model()) == "ceos" `)